docs/ref/checks.txt

```diff
@@ -750,9 +750,6 @@ The following checks are performed on the default
   must be in :setting:`MIDDLEWARE` in order to use the admin application.
 * **admin.E410**: :class:`django.contrib.sessions.middleware.SessionMiddleware`
   must be in :setting:`MIDDLEWARE` in order to use the admin application.
-* **admin.W411**: ``django.template.context_processors.request`` must be
```

Comment:
When removing a check, we should preserve a record of the code somewhere so it doesn't get reused in the future (as it could already be used in projects' `SILENCED_SYSTEM_CHECKS`).

Reply:
That makes sense, thanks. Do you know if there is an established approach to this? Should I just leave the line as is? Move it somewhere else? Or modify it some some marker?

Reply:
I added `*This check appeared in Django 3.1*.` which I saw done for other system checks.

Reply:
Yeah it looks like that's the mechanism right now.